### ISSUE TITLE
clojure-fill-paragraph doesn't wrap backwards

### ISSUE THREAD

**Issue by arrdem** [clojure-emacs/clojure-mode#400]
At present, `clojure-fill-paragraph` only scans forwards to wrap lines. I argue that it should scan backwards to the first empty comment line or start of comment and fill everything from that point down.

So for instance: at present

``` clojure
;; foo bar baz
;; qux|
```

fill is a noop.

Instead, it should fill to

``` clojure
;; foo bar baz qux|
```

However:

``` clojure
;; foo bar baz
;;
;; quixxity quuxxity
;; flub|
```

should fill to

``` clojure
;; foo bar baz
;;
;; quixxity quuxxity flub|
```

**Comment by vspinu** [comment]
Fixed by #434 btw.

**Comment by arrdem** [comment]
Thanks!
